timber/src/main/java/timber/log/Timber.java

```diff
@@ -24,6 +24,13 @@ public static void v(Throwable t, @NonNls String message, Object... args) {
     TREE_OF_SOULS.v(t, message, args);
   }
 
+  /** Log a verbose exception and a message without the StackTrace
+   * with optional format args,  */
+  public static void v(Throwable t, @NonNls String message,
+                       Boolean addStackTrace, Object... args) {
```

Comment:
It is not nice to separate message from its args, and the flag from its throwable